### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by queengooborg**
This PR updates the package.json to populate additional properties like in `translated-content`.  Although it's not really important, it's a nice-to-have tweak.

### CODE REVIEW

File: yarn.lock
Comment (bsmth) [inline yarn.lock:1]: Did you run `yarn upgrade` here?

Comment (queengooborg) [reply yarn.lock:1]: Ack- thought I reverted these changes, whoops!

File: package.json [hunk 1]
```diff
@@ -1,8 +1,18 @@
 {
-  "name": "content",
+  "name": "@mdn/content",
```
Comment (bsmth) [inline package.json:2]: Are there any side-effects for consumers by this scoping?

We scope it this way in yari:

https://github.com/mdn/yari/blob/86a4e6fbaaa1ad85a8c47c065f4c15eb4e6d4efc/package.json#L2

Comment (queengooborg) [reply package.json:2]: Nope, no side effects, since this package is private and is never imported or published as a package!

**Review by bsmth — APPROVED** [review]
Thanks @queengooborg - I'm going to leave a +1 on this 👍🏻

**Pull request merged**